tech/languages/ruby/ror-installation.md

```diff
@@ -8,11 +8,11 @@ order: 4
 
 ## Installing Rails from RubyGems.org
 
-To install Ruby on Rails on Fedora as a gem, install Ruby first together with `ruby-devel`, `gcc`, `libxml2-devel` packages, and then install Rails using the `gem` command:
+To install Ruby on Rails on Fedora as a gem, install Ruby first together with `ruby-devel`, `gcc`, `zlib-devel` packages, and then install Rails using the `gem` command:
```

Comment:
> Hello, I have tested the original code with Rails 5.0.0.1 and it works fine. Could you please elaborate on the change?

If the test works fine with original code, you think it is correct?

I think that the document describes minimal steps to install "upstream" rails.
Maybe it is better and suitable to install upstream rails with libxml2 that is included in upstream rails, than RPM libxml2.